src/pages/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,109 @@
+import { useState } from "react";
+import { useUser } from "../../contexts/UserProvider";
+import { User } from "../../data/models/User";
+
+import EntriesIcon from "../../assets/book.svg";
+import JournalIcon from "../../assets/encrypted.svg";
+import StatsIcon from "../../assets/graph.svg";
+import GroupsIcon from "../../assets/groups.svg";
+import SettingsIcon from "../../assets/settings.svg";
+import LinkButton from "../../components/LinkButton";
+import HappinessForm from "./HappinessForm";
+import { useWindowDimensions } from "../../utils";
+
+export default function Sidebar({ element }: { element: React.ReactElement }) {
+  const { user } = useUser();
+
+  const navConfig = [
+    { title: "Entries", route: "/home", icon: EntriesIcon },
+    { title: "Private Entries", route: "/journal", icon: JournalIcon },
+    { title: "Stats", route: "/statistics", icon: StatsIcon },
+    { title: "Groups", route: "/groups", icon: GroupsIcon },
+    { title: "Settings", route: "/settings", icon: SettingsIcon },
+  ];
+
+  const [select, setSelect] = useState("Entries");
+
+  const { height } = useWindowDimensions();
+
+  return (
+    <>
+      <div className="flex w-full">
+        <div
+          id="docs-sidebar"
+          className="scrollbar-y dark:scrollbar-y dark:bg-gray-800 dark:border-gray-700 bottom-0 left-0 top-0 z-[60] hidden min-w-[320px] max-w-[320px] transform overflow-y-auto border-gray-200 bg-light_yellow transition-all duration-300 lg:bottom-0 lg:right-auto lg:block lg:translate-x-0"
+        >
+          <div className="flex h-full flex-col">
+            <div className="m-4 flex grow flex-col">
+              <div className="m-4 flex grow flex-col">
+                <div className="mb-1 text-sm font-semibold text-black">
+                  Account
+                </div>
+                <a
+                  className="flex-none dark:text-white"
+                  href={"/profile/" + user!.id}
+                >
+                  <div className="mb-6 flex items-center rounded-xl">
+                    <div
+                      className="mr-2 items-center"
+                      onClick={() => setSelect("")}
+                    >
+                      <img
+                        className="mx-3 mx-auto block max-h-[40px] max-w-[40px] justify-center rounded-full sm:mx-0 sm:shrink-0"
+                        src={user!.profile_picture}
+                        alt="profile"
+                      />
+                    </div>
+                    <div className="mr-2 text-secondary">
+                      <div className="text-base font-semibold">
+                        {user!.username}
+                      </div>
+                      <div className="text-xs text-light_gray">
+                        Logging since {user!.created.substring(0, 4)}
+                      </div>
+                    </div>
+                    <button
```

Comment:
This button shouldn't have a drop shadow (see design). But actually don't worry about this because I will be refactoring this to use my generic `Button` component anyway. But just pointing this out, it's important to closely follow the design or the app feel will be off.

Reply:
Wait, doesn't the design say that there is a box shadow?
![image](https://github.com/jonathanjma/happiness-app-frontend/assets/67171896/fb17fed6-c7bb-435b-a588-97bd5521c9e6)

Reply:
yeah but if you look at the Figma the shadow definitely isn't showing up on the actual Figma. I can ask design about this, it's weird that Figma shows the shadow style but then it doesn't actually look like a shadow on Figma. Also looking at these side by side the button label should be bold so could you fix that?

Figma version:
![image](https://github.com/jonathanjma/happiness-app-frontend/assets/58796478/490c4d49-4408-4449-8f00-01e0db2a29a7)

Your version with shadow:
![image](https://github.com/jonathanjma/happiness-app-frontend/assets/58796478/fc5093cd-30ee-4e15-8d61-b8798caa11f1)